docker/files/systemd.conf

```diff
@@ -22,6 +22,7 @@ Restart=always
 ExecStart=/usr/bin/docker run {% for option in runoptions %}{{ option }} {% endfor %} --name={{ name }} {{ container.image }} {{ cmd }}
 ExecStop=/usr/bin/docker stop {{ name }}
 ExecStopPost=/usr/bin/docker rm -f {{ name }}
+StandardOutput=null
```

Comment:
Just out of curiosity, do you think this should be absolutely mandatory by default?
As for me, it is very handy to see some logs in `systemctl status ${CONTAINER}` output. At least this could be made configurable from Pillar.

Reply:
Actually these salt states create systemd service  named `docker-${CONTAINER}`
so you can `not` see logs via `systemctl status ${CONTAINER}`

problem is that docker *by default* writes to `/var/lib/docker/containers/*/*-json.log`
and you can easily view them with `docker logs ${CONTAINER}`
Also logs collectors sucj as fluentd, filebeat read logs directly from /var/lib/docker/containers

Systemd service gets the same logs and writes them to `/var/log/messages`

As for me duplicating logs from containers are useless here especially in cases of intensive logs  - where you can quickly get out of disk space on host machine